pkg/sfu/buffer/buffer.go

```diff
@@ -66,14 +62,11 @@ type Buffer struct {
 	latestTSForAudioLevel            uint32
 
 	lastPacketRead int
-	bitrate        atomic.Value
-	bitrateHelper  [4]int64
```

Comment:
All the bitrate calculation moves to `StreamTrackerManager` which can be re-used in relay also.